src/App/Resources/AppResources.resx

```diff
@@ -518,23 +518,29 @@
   <data name="Lock" xml:space="preserve">
     <value>Lock</value>
   </data>
-  <data name="LockOption15Minutes" xml:space="preserve">
+  <data name="VaultTimeout15Minutes" xml:space="preserve">
     <value>15 minutes</value>
   </data>
-  <data name="LockOption1Hour" xml:space="preserve">
+  <data name="VaultTimeout1Hour" xml:space="preserve">
     <value>1 hour</value>
   </data>
-  <data name="LockOption1Minute" xml:space="preserve">
+  <data name="VaultTimeout1Minute" xml:space="preserve">
     <value>1 minute</value>
   </data>
-  <data name="LockOption4Hours" xml:space="preserve">
+  <data name="VaultTimeout4Hours" xml:space="preserve">
     <value>4 hours</value>
   </data>
-  <data name="LockOptionImmediately" xml:space="preserve">
+  <data name="VaultTimeoutImmediately" xml:space="preserve">
     <value>Immediately</value>
   </data>
```

Comment:
Should we just call these 15Minutes, 1Hour, etc and drop the prefix altogether?

Reply:
I don't really have a problem with that since they don't needed have any real dependency with the VaultTimeout prefix.